lib/active_merchant/billing/gateways/alelo.rb

```diff
@@ -4,7 +4,7 @@ module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class AleloGateway < Gateway
       self.test_url = 'https://sandbox-api.alelo.com.br/alelo/sandbox/'
-      self.live_url = 'https://desenvolvedor.alelo.com.br'
+      self.live_url = 'https://api.alelo.com.br/alelo/prd/'
```

Comment:
should we include a variable with the homologation URL?  like self.pre_live

Reply:
That would be good - we do this for some other gateways like `litle`

Reply:
Done